ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+
+== Reasoning
+
+=== Based on Python PEP, TAPS, and JEP
+
+The process ITE is modeled after the python, TUF, and Jenkins communities, who
+have successfully navigated several large-scale reworkings of their codebases,
+specifications and processes over the past decade. This includes most notably
+the multi-year project of Python 3 (formerly Python 3000) and key format
+changes after security audits, as it is in the case of TUF.
+
+Their Python Enhancement Proposals are largely consensus driven, which is
+_mostly_ how work is done presently in the in-toto project, making the PEP
+model relatively straightforward to graft onto our existing, processes for
+making proposals and deciding upon changes.
+
+The TUF Augmentation Proposal model is a closer relative to ITE's due that
+in-toto and TUF are sister projects. Given the closeness of the communities and
+their overlapping developer base, trying to maintain similarities with both
+processes is paramount.
+
+Finally, the Jenkins Enhancement Proposals were the main inspiration of this
+document, as JEP 01 was perceived as clearer, more thorough and less ambiguous
+than their Python and TUF counterparts.
+
+=== Differences from Jenkins's JEP
+
+The Jenkin's JEP proposes a BDFL figure as the ultimate source conflict
+resolution. Due to the horizontal nature of in-toto, this aspect of JEP 01 was
+omitted.
+
+=== Differences from Python PEP
+
+The Python process uses "Rationale" as the heading for the section for
+describing design decisions.  However the meaning of "Rationale" is similar to
+"Motivation" in some contexts.  We decided to use "Reasoning" instead to avoid
+confusion.
+
+=== Differences from TUF's TAPs
+
+The TAP document is closer to a PEP than ITEs are, for ITE's adopted the
+organizational structure of JEPs. Furthermore, ITE's adopted asciidoc rather
+than markdown, as its syntax has less derivate dialects (e.g., github flavored)
+than markdown does. Furtermore, the repository structure of JEP was adopted, as
+it allows for the addition of self-hosted diagrams and supporting documents.
+
+=== Asciidoc style and linter
+
+There are a number of possible asciidoc style guidelines.
+For example, there are a number of reasons to use
+link:http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line[one sentence per line],
+one phrase per line, or other specific formatting.
+However, choosing which guideline to require, recommend, or even make optional,
+is a potentially long and difficult process.
+Instead of bogging down this ITE in that process,
+we decided to consider asciidoc style and formatting guidelines in a later ITE.
+
+== Backwards Compatibility
+
+There are no backwards compatibility concerns related to this proposal.
+
+== Security
+
+There are no security risks related to this proposal.
+
+== Infrastructure Requirements
+
+There are no new infrastructure requirements related to this proposal.
+This ITE leverages existing infrastructure.
+
```

Comment:
Do you need to include a "== Testing section", and say this ITE has no testing needs? (based on the earlier definitions)

Reply:
I think that's implied that since this doesn't include code testing isn't included. Would you want me to rephrase the definition or add the header to reflect this?

Reply:
Earlier in this document, under "ITE Guidelines", "Required Sections", you say that for **Testing**:
"If the ITE has no testing needs, the section must say that."

Similarly, for **Backwards Compatibility**, you say that:
"If there are no backwards compatibility concerns, the section must say that."

And for **Security**:
"If the ITE has no impact on security, the section must say that."

So, I see that for this ITE, you DID include sections for **Security** and **Backwards Compatibility**, which say that there is no impact, but you did not do that for **Testing**.

Reply:
right. Sorry I'll fix this